packages/app/src/runner/events/capture-protocol.ts

```diff
@@ -79,7 +79,9 @@ export const addCaptureProtocolListeners = (Cypress: Cypress.Cypress) => {
       timestamp: performance.now() + performance.timeOrigin,
     })
 
-    await Cypress.backend('protocol:test:before:run:async', attributes)
+    const params = { ...attributes, config: { testIsolation: !!Cypress.config('testIsolation') } }
```

Comment:
I think we actually need to pass in whether we're doing component testing as well here. It has similar things to work through.

Reply:
Since component testing is for the entire run, I wonder if that should be passed in somewhere else.

Reply:
Updated `setupProtocol` to pass the `testingType`. [7b24eab](https://github.com/cypress-io/cypress/pull/27523/commits/7b24eabe1c3a7618e7efbb4f56c32bbb3b4b594f)